definitions/features/foreman_server.rb

```diff
@@ -41,6 +47,16 @@ def config_files_to_exclude
           '/var/lib/foreman/public'
         ]
       end
+
+      def services_running?
+        services.all?(&:running?)
+      end
+
+      private
+
+      def foreman_service_installed?
+        package_manager.installed?(['foreman-service']) && check_min_version('foreman', '2.1')
```

Comment:
I have kept code as it is because through foreman-maintain user can perform all service actions i.e disable, enable, start, stop, restart, list. So will the use of `systemctl is-enabled foreman` be helpful here?

Reply:
The installer has supported deploying with foreman-service since 1.23 using `--foreman-passenger false`. With 2.1 we're making it the default. 2.1 also introduces `foreman.socket`. Prior to that `foreman.service` (and earlier the sysvinit version) were shipped as part of Foreman itself, dating back to as far as I can remember. IMHO `systemctl is-enabled foreman` is a good check. The installer will ensure it's either enabled or disabled. IIRC `is-enabled` also returns the correct value if a service is missing.